cms/apps/media/templates/admin/media/includes/focal-point.html

```diff
@@ -0,0 +1,79 @@
+<fieldset class="module aligned with-legend ">
+  <h2 class="legend">Focal point</h2>
+  <div>Specifying a focal point for an image should prevent that location from being cropped.</div>
+  <div id="focal-point-clear">Clear</div>
+
+  <div id="focal-point-container" style="position: relative; width: fit-content">
+    <img id="focal-point" style="filter:{% if original.focal_y %}brightness(50%){% else %}brightness(100%){% endif %};" src="{{ original.get_absolute_url }}"/>
+    <div id="focal-point-pointer"></div>
+  </div>
+  <script>
+    (function () {
+      document.addEventListener('DOMContentLoaded', function() {
+        document.getElementById('focal-point').addEventListener('click', function (e) {
+          var pointer = document.getElementById('focal-point-pointer')
+          var image = document.getElementById('focal-point')
+
+          var imageW = image.offsetWidth;
+          var imageH = image.offsetHeight;
+
+          //Calculate FocusPoint coordinates
+          var offsetX = e.pageX - image.getBoundingClientRect().left;
+          var offsetY = e.pageY - image.getBoundingClientRect().top;
```

Comment:
Are you sure this is not going to miscalculate? [`mouseEvent.pageY`](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/pageY) "returns the Y (vertical) coordinate in pixels of the event relative to the whole document". `getBoundingClientRect().top` returns the Y position relative to the viewport.